src/pipecat/services/resembleai/tts.py

```diff
@@ -0,0 +1,286 @@
+#
+# Copyright (c) 2024–2025, Daily
+#
+# SPDX-License-Identifier: BSD 2-Clause License
+#
+
+import base64
+import json
+from typing import AsyncGenerator, Optional
+
+from loguru import logger
+
+from pipecat.frames.frames import (
+    CancelFrame,
+    EndFrame,
+    ErrorFrame,
+    Frame,
+    StartFrame,
+    StartInterruptionFrame,
+    TTSAudioRawFrame,
+    TTSStartedFrame,
+    TTSStoppedFrame,
+)
+from pipecat.processors.frame_processor import FrameDirection
+from pipecat.services.tts_service import AudioContextWordTTSService
+from pipecat.utils.text.base_text_aggregator import BaseTextAggregator
+from pipecat.utils.text.skip_tags_aggregator import SkipTagsAggregator
+
+try:
+    import websockets
+except ModuleNotFoundError as e:
+    logger.error(f"Exception: {e}")
+    raise Exception(f"Missing module: {e}")
+
+
+class ResembleTTSService(AudioContextWordTTSService):
+    def __init__(
+        self,
+        *,
+        api_key: str,
+        voice_uuid: str,
+        url: str = "wss://websocket.cluster.resemble.ai/stream",
+        sample_rate: Optional[int] = 32000,
+        output_format: str = "wav",
+        precision: str = "PCM_16",
+        binary_response: bool = False,
+        no_audio_header: bool = True, # avoid audio header included in the output
+        text_aggregator: Optional[BaseTextAggregator] = None,
+        **kwargs,
+    ):
+        # Aggregating sentences still gives cleaner-sounding results and fewer
+        # artifacts than streaming one word at a time. On average, waiting for a
+        # full sentence should only "cost" us 15ms or so with GPT-4o or a Llama
+        # 3 model, and it's worth it for the better audio quality.
+        # We also don't want to automatically push LLM response text frames,
+        # because the context aggregators will add them to the LLM context even
+        # We can use those to generate text frames ourselves aligned with the
+        # playout timing of the audio!
+        super().__init__(
+            aggregate_sentences=True,
+            push_text_frames=False,
+            pause_frame_processing=True,
```

Comment:
Need this for the base class.

```suggestion
            push_stop_frames=True
            pause_frame_processing=True,
```